README.md

```diff
@@ -113,18 +113,18 @@ You should now see a `debug` command next to the `run` command in the Gradle Pro
 </details>
 <details><summary>Pin tasks</summary>
 
-As there could be many tasks in a Gradle project, it can be useful to pin commonly used tasks. Pinned tasks will be shown in a seperate view. Pin a task by accessing the task context menu (by right-clicking a task). You can also pin a task with specific arguments.
+As there could be many tasks in a Gradle project, it can be useful to pin commonly used tasks. Pinned tasks will be shown under their project item. Pin a task by accessing the task context menu (by right-clicking a task). You can also pin a task with specific arguments.
 
 <img src="./images/pin-task.png" width="350" alt="Pin a Gradle Task" />
 
-To remove a pinned a task, access the task context menu and select `Remove Pinned Task`, or clear all pinned tasks by clicking on the `Clear Pinned Tasks` button in the treeview header.
+To remove a pinned a task, access the task context menu and select `Remove Pinned Task`, or clear all pinned tasks by clicking on the `Clear Pinned Tasks` button in the overflow button of tree view header.
```

Comment:
Just curious about how other IDEs call this action, I recall `unpin task` somewhere?

Reply:
Got it, vscode use the name "unpin".
![image](https://user-images.githubusercontent.com/2351748/155175073-c70455d5-9af0-4402-8fe2-115962a30e52.png)

Reply:
Let's rename it from `Remove pinned task` to `Unpin task`.